coverage_comment/settings.py

```diff
@@ -29,6 +29,7 @@ class Config:
     GITHUB_TOKEN: str = dataclasses.field(repr=False)
     GITHUB_REPOSITORY: str
     GITHUB_REF: str
+    GITHUB_OUTPUT: str
```

Comment:
```suggestion
    GITHUB_OUTPUT: pathlib.Path | None = None
```

See `COMMENT_FILENAME` below and `clean_comment_filename` below for a way to handle this as a `Path()` instead of a string. This will make the rest of the code easier.

I'm also suggesting adding a default value to None (so you'll need to move this line a few lines down, because you can't have attribute with default values above attributes without default values)